x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch

```diff
@@ -0,0 +1,36 @@
+# Build fails with New clang 16.0 compiler because it finds a new error -Wenum-constexpr-conversion
+# Upstream issue: https://gitlab.gnome.org/GNOME/vte/-/issues/2618
+# Patch authored by Khem Raj
```

Comment:
Please always put the commit link in the patch too.

Also, to preserve metadata, append .patch to the URL to the commit and use that as the basis for the patch.

Reply:
Doing it.

Reply:
It's got the link but you haven't included all the metadata at the top?

Reply:
My bad, is it alright now?